demo/demo.md

```diff
@@ -28,58 +28,103 @@ To meet our accessibility requirement, all uses of `auro-dropdown` should have a
 1. Use `aria-label` to insert label content that will only be read by screen readers
 1. Use `aria-labeledby` to append a description from another element on the page
 
-Not including one of the above options will result in your UI being non-compliant with Alaska's accessibility policies.
-
+Not including one of the above options will result in your UI being non-compliant with Alaska's accessibility policies.
+
 ## Common use with auro-label
 
 This first common example uses the default `auro-dropdown` element with the attributes of `bordered` `rounded` `inset` `toggle` and `chevron`. Additionally the `aria-label` attribute is used to define a string value that labels an interactive element.
 
```

Comment:
This content is the same header content from lower in the page? This doesn't speak to the updated API or why these examples are here?

How is the `modalTrigger` API used?